qiita_pet/templates/study_ajax/prep_summary.html

```diff
@@ -1,115 +1,401 @@
+      }
+      else {
+        // It is a user defined value
+        $("#user-ena-info").show();
+        $("#ena-ontology").val('Other');
+        $("#user-ontology").val('{{investigation_type}}');
+      }
+    }
+
+    load_information_summary();
+    // If the template already have an artifact attached, load the
+    // artifact graph, otherwise load the page to add a new artifact
+    if ("{{artifact_attached}}" == "True") {
+      load_template_graph();
+    }
+    else {
+      load_new_artifact();
+    }
+    // If the file-selector select changes, check if we need to show the update button
+    $("#file-selector").change(function ( event ) {
+      if ( $("#file-selector").val() === "" ) {
+        $("#update-button-div").hide();
+      }
+      else {
+        $("#update-button-div").show();
+      }
+    });
+
+    // If the ena-ontology select changes, check if we need to show the user list
+    $("#ena-ontology").change(function ( event ) {
+      var value = $("#ena-ontology").val();
+      if( value == "Other" ) {
+        $("#user-ena-info").show();
+        $("#user-ena-info").trigger('change');
+      }
+      else {
+        $("#user-ena-info").hide();
+        $("#new-ena-info").hide();
+        update_investigation_type(value);
+      }
     });
+    // If the user-ena-info select changes, check if we need to show the input box
     $("#user-ena-info").change(function ( event ) {
-        if( $("#user-ontology").val() == "New Type" ) { $("#new-ena-info").show(); }
-        else { $("#new-ena-info").hide(); }
+        var value = $("#user-ontology").val();
+        if( value == "New Type" ) {
+          $("#new-ena-info").show();
+        }
+        else {
+          $("#new-ena-info").hide();
+          if( value !== "" ) {
+            update_investigation_type(value);
+          }
+        }
     });
   });
-</script>
\ No newline at end of file
+</script>
+
+<!-- Prep template title and buttons -->
+<div class="row">
+  <div class="col-md-12">
+    <h4>
+      {{name}} - ID {{prep_id}}
+      <a class="btn btn-default glyphicon glyphicon-download-alt" href="/download/{{download_prep}}" style="word-spacing: -10px;"> Prep info</a>
+      <a class="btn btn-default glyphicon glyphicon-download-alt" href="/download/{{download_qiime}}" style="word-spacing: -10px;"> Qiime map</a>
+      <a class="btn btn-danger glyphicon glyphicon-trash" style="word-spacing: -10px;" onclick="delete_prep_info({{prep_id}});"> Delete</a>
+    </h4>
+  </div>
+</div>
+
+<!-- #samples and #columns -->
+<div class="row">
+  <div class="col-md-12">
+    There are <b>{{num_samples}}</b> samples and <b>{{num_columns}}</b> columns in this preparation.
+  </div>
+</div>
+
+<!-- Update prep template -->
+<div class="row">
+  <div class="col-md-12">
+    <b>Update information:</b>
+    <select id="file-selector">
+      <option value="">Choose file...</option>
+      {% for fp in files %}
+        <option value="{{fp}}">{{fp}}</option>
```

Comment:
I think space will do it, which is allowed by the upload. We could change this behavior in the uploader. Wanna open an issue?